src/hotspot/cpu/riscv/macroAssembler_riscv.cpp

```diff
@@ -531,7 +531,7 @@ void MacroAssembler::resolve_jobject(Register value, Register thread, Register t
   beqz(value, done);           // Use NULL as-is.
 
   // Test for jweak tag.
-  andi(t0, value, JNIHandles::weak_tag_mask);
+  test_bit(t0, value, exact_log2(JNIHandles::weak_tag_mask));
```

Comment:
I don't think this is a single-bit testing, right?

Reply:
Thanks for the review! I think this is a single-bit testing and `JNIHandles::weak_tag_mask` is defined here:
https://github.com/openjdk/riscv-port-jdk17u/blob/cc8f3f9e84f7c926f1a793651c001cc5b3f945c9/src/hotspot/share/runtime/jniHandles.hpp#L70-L72